test/integration/relay-analytics/pages/_app.js

```diff
@@ -0,0 +1,8 @@
+/* global localStorage */
+import App from 'next/app'
+
+export default class MyApp extends App {}
+
+export function relayPerformanceData (data) {
```

Comment:
Not sure I'm fond of this name -- can we call it `on` instead of `relay`?

Reply:
oh yes, i always love suggestions on the name

Reply:
changed to `onPerformanceData`